lib/jobs/create-default-pollers.js

```diff
@@ -83,7 +83,7 @@ function createDefaultPollersJobFactory(
      */
     CreateDefaultPollersJob.prototype.createUcsPoller = function(nodeIds, poller){
         var self = this;
-        return Promise.map(nodeIds, function(nodeId) {
+        return Promise.each(nodeIds, function(nodeId) {
```

Comment:
Is there any reason to change it from map to each?

Reply:
Please refer to: https://rackhd.atlassian.net/browse/RAC-6140